src/form-engine.scss

```diff
@@ -36,32 +36,35 @@
   overflow-y: hidden;
   display: flex;
   flex-direction: column;
+  justify-content: space-between;
 }
 
-.formContentBody {
+.contentBody {
   overflow-y: auto;
   width: inherit;
   position: sticky;
 }
 
-.minifiedFormContentBody {
-  margin-bottom: 4rem !important;
-}
-
 .minifiedButtons {
-  width: 100%;
-  height: 63.5px;
-  position: absolute;
-  bottom: 0;
+  button {
+    height: 4rem;
+    display: flex;
+    align-content: flex-start;
+    align-items: baseline;
+    min-width: 50%;
+  }
 }
 
-.minifiedButtons :global(.cds--btn) {
-  max-width: 50%;
-  height: 3.9rem;
-}
+:global(.omrs-breakpoint-lt-desktop) {
+  .content {
+    height: var(--tablet-workspace-window-height);
+  }
 
-.minifiedButtons > button {
-  width: 50%;
+  .minifiedButtons {
+    border-top: 1px solid #ededed;
```

Comment:
@pirupius this should help assuage the concern you had:

![CleanShot 2024-05-21 at 11  44 08@2x](https://github.com/openmrs/openmrs-form-engine-lib/assets/8509731/1adb4624-a37a-4d24-a585-d270f4f51dce)

Reply:
Thanks @denniskigen